storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3781,6 +3791,24 @@ class Rdb_writebatch_impl : public Rdb_transaction {
     return get(column_family, key, value);
   }
 
+  void multi_get(const rocksdb::ReadOptions &read_options,
+                 rocksdb::ColumnFamilyHandle *column_family,
+                 const size_t num_keys, const rocksdb::Slice *keys,
+                 rocksdb::PinnableSlice *values, rocksdb::Status *statuses,
+                 bool sorted_input) override {
+    // todo: could we just read the committed content from the DB here?
```

Comment:
Nothing is wrong, and I think we can do with just one `multi_get` function now (since the bypass' function switched to using the same function signature.